webapi/src/main/scala/org/knora/webapi/responders/v2/ResourceUtilV2.scala

```diff
@@ -133,25 +134,25 @@ object ResourceUtilV2 {
     } yield defaultObjectAccessPermissionsResponse.permissionLiteral
 
   /**
-   * Checks whether a list node exists, and throws [[NotFoundException]] otherwise.
+   * Checks whether a list node exists and has a root node, otherwise throws [[NotFoundException]].
    *
    * @param listNodeIri the IRI of the list node.
    */
-  def checkListNodeExists(listNodeIri: IRI, appActor: ActorRef)(implicit
+  def checkListNodeExistsAndHasRootNode(listNodeIri: IRI, appActor: ActorRef)(implicit
```

Comment:
also, I'm missing a test for this method.

Reply:
That wasn't we but you are trying to force me to do so :) My opinion was different on that. However I've tried and spent some time to find out, that this method has been extracted to `ResourceUtilV2` because it's used twice in `ValuesResponderV2` as well as once in `ResourcesResponderV2`, where actually the usage is slightly different and I couldn't make things compile. Changing it from Unit method which throws to returning Boolean will also create more code duplication.
All this only strengthen my feelings:
1. I'm against increasing the scope of tasks, especially bugs. If we have bugs first policy, they should be fixed ASAP. This task is about fixing the bug, which I did applying the convention in the package, I also wrote test.
2. Changing  the way how only `checkListNodeExistsAndHasRootNode` works w/o touching other methods in `ResourceUtilV2` doesn't bring any value, in fact it makes even more confusion. If this is so important - should be moved to refactor task which will solve the problem comprehensively.
4. At the end I don't mind to refactor code while fixing the bugs, which in fact we all do. But this should be rather only necessary changes which helps in fixing the bug, like in this PR changing `checkListNodeExists` to `checkListNodeExistsAndHasRootNode` in order to reduce number of queries and lines of code - one method instead of two.